newrelic/exporter.go

```diff
@@ -78,13 +79,18 @@ func (e *Exporter) ExportSpan(ctx context.Context, span *tracesdk.SpanData) {
 }
 
 // Export exports metrics to New Relic.
-func (e *Exporter) Export(_ context.Context, r *resource.Resource, cps metricsdk.CheckpointSet) error {
-	return cps.ForEach(func(record metricsdk.Record) error {
-		m, err := transform.Record(e.serviceName, r, record)
+func (e *Exporter) Export(_ context.Context, cps metricsdk.CheckpointSet) error {
+	return cps.ForEach(e, func(record metricsdk.Record) error {
+		m, err := transform.Record(e.serviceName, record.Resource(), record)
 		if err != nil {
 			return err
 		}
 		e.harvester.RecordMetric(m)
 		return nil
 	})
 }
+
+func (e *Exporter) ExportKindFor(_ *metric.Descriptor, _ aggregation.Kind) metricsdk.ExportKind {
+	// TODO: Check if this exporter kind is correct
+	return metricsdk.PassThroughExporter
```

Comment:
This should probably be a `DeltaExporter` as all of our metrics expect a delta value.

```suggestion
	return metricsdk.DeltaExporter
```

Reply:
Thank you for checking, fixed to `DeltaExporter`